app-backend/batch/src/main/scala/cogMetadata/OverviewBackfill.scala

```diff
@@ -0,0 +1,78 @@
+package com.rasterfoundry.batch.cogMetadata
+
+import com.rasterfoundry.batch.Job
+import com.rasterfoundry.common.{AWSLambda, RollbarNotifier}
+import com.rasterfoundry.datamodel.ProjectLayer
+import com.rasterfoundry.database.ProjectLayerDao
+import com.rasterfoundry.database.Implicits._
+import com.rasterfoundry.database.util.RFTransactor.xaResource
+
+import cats.effect.IO
+import doobie._
+import doobie.implicits._
+import doobie.postgres.implicits._
+import fs2.Stream
+
+object OverviewBackfill extends Job with RollbarNotifier {
+  val name = "backfill-layer-overviews"
+
+  def getProjectLayerSceneCount(
+      projectLayer: ProjectLayer
+  ): Stream[ConnectionIO, (ProjectLayer, Int)] =
+    fr"select count(1) from scenes_to_layers where project_layer_id = ${projectLayer.id}"
+      .query[Int]
+      .stream map { (projectLayer, _) }
+
+  /** We know suppressing this warning is fine, because the query for project layers
+    * explicitly filters out any that don't have a project id
+    */
+  @SuppressWarnings(Array("OptionGet"))
+  def kickoffOverviewGeneration(
+      projectLayer: ProjectLayer
+  ): IO[Unit] = IO {
+    logger.info(
+      s"Kicking off lambda overview generation for layer ${projectLayer.id}")
+    if (AWSLambda.runLambda) {
+      AWSLambda
+        .kickoffLayerOverviewCreate(
+          projectLayer.projectId.get,
+          projectLayer.id,
+          "RequestResponse"
+        )
+    } else {
+      logger.info("Sleeping for 10 seconds to pretend to do work")
+      Thread.sleep(10000)
```

Comment:
This exists to show the limited concurrency in parEvalMap -- it doesn't need to stick around but also won't hurt anything and makes local parallelism behave more sensibly than not doing any work at all